spec/components/hijack_spec.rb

```diff
@@ -58,7 +58,7 @@ def hijack_test(&blk)
         end
       end
 
-      env = {}
+      env = { "rack.input" => StringIO.new }
```

Comment:
This seems to be unrelated to the changes in this PR?

Reply:
I added it to make a spec pass when I was working on the old implementation, but confirm if it's still needed.

Reply:
Removed it.